lib/lotus/commands/destroy.rb

```diff
@@ -0,0 +1,73 @@
+module Lotus
+  module Commands
+    class Destroy
+
+      ACTION_SEPARATOR = /\/|\#/
+
+      attr_reader :name, :app_name
+
+      def initialize(type, app_name, name, env, cli)
```

Comment:
ideally, we would have many versions of destroyers, can we replicate same pattern like those of generators, that is having `destroyers/action.rb`